package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -151,14 +151,16 @@ preload_rke2_images()
     RKE2_IMAGES_LISTS_DIR="/tmp/images-lists"
     mkdir -p $TARGET/$RKE2_IMAGES_DIR
     mkdir -p $TARGET/$RKE2_IMAGES_LISTS_DIR
-    mount --bind ${ISOMNT}/bundle/harvester/images $TARGET/$RKE2_IMAGES_DIR
     mount --bind ${ISOMNT}/bundle/harvester/images-lists $TARGET/$RKE2_IMAGES_LISTS_DIR
 
+    echo "Copying RKE2 images to the target location..."
+    rsync -ah --progress ${ISOMNT}/bundle/harvester/images/rke2-images.*.tar.zst $TARGET/$RKE2_IMAGES_DIR
```

Comment:
The `rke2*` images are still copied to the same path because `rke2 server` needs them. Other images are copied to a different location to prevent `rke2 server` from preloading them automatically.